spectrochempy/core/readers/importer.py

```diff
@@ -206,7 +208,7 @@ def _switch_protocol(self, key, files, **kwargs):
                     datasets.extend(res)
 
             except FileNotFoundError:
-                raise
+                warning_(f"No file with name `{filename}` could be found. Sorry! ")
 
```

Comment:
This change can explain the error you get in test_importer. May be issue the warning but raise an error to avoid breaking the previous behaviour.

Reply:
Yes. I've chosen the wrorng option in the merge conflict...
I will revert this